brianlong/www.validators.app
# 184297117 ignore stake_accounts with stake smaller than minimum

#### What's this PR do?
- Skips creating StakeAccounts with small stake (smaller than 1 SOL atm)

#### How should this be manually tested?
On staging:
- Push to staging and wait until the script completes (it's triggered 10 minutes after each hour)
- Compare results on https://stage.validators.app/stake-pools?locale=en&network=mainnet against production 
- Compare pool stats (eg. on BlazeStake the number of nodes should drop to about 107, average stake should go up, etc.).
- All stake accounts with stake smaller than 1 SOL should disappear (see examples on a screenshot below). 

Locally:
- Run `rails r script/gather_stake_accounts.rb` on master. Check the number of StakeAccounts ( eg. `StakeAccount.where(network: 'mainnet').count` )
- Run gather+score daemons to generate a new batch.
- Run gather_stake_accounts script on this branch. Check the number of StakeAccounts now: `StakeAccount.where(network: 'mainnet').count`
- Run server and compare http://localhost:3000/stake-pools?locale=en&network=mainnet view against production. Stake Accounts with stake smaller than 1 SOL should now disappear.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184297117)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

<img width="1036" alt="Zrzut ekranu 2023-01-23 o 21 15 50" src="https://user-images.githubusercontent.com/32059143/214141194-2ec2fcd9-9601-4e65-b431-d87d559fee70.png">
